README.md

```diff
@@ -101,7 +101,7 @@ zoxide init fish | source
 
 ### Environment variables
 
-- `$_ZO_DATA`: sets the location of the database (default: `~/.zo`)
+- `$_ZO_DATA_DIR`: directory where `zoxide` will store its data files (default: `$HOME/.local/share/zoxide`)
```

Comment:
`$HOME/.local/share/zoxide` isn't really accurate since it's only true for Linux systems. I don't have the answer to what you would replace it with without being needlessly verbose, though.

Reply:
We could just cut out the "default" verbiage, if you're fine with it. Otherwise, I'll brainstorm something better.

Reply:
That seems fine to me.

Reply:
I actually changed the "default:" wording to point to the `dirs` docs. Is this a satisfactory solution, or are you still leaning more towards dropping it altogether?